com.unity.render-pipelines.high-definition/Documentation~/Dynamic-Resolution.md

```diff
@@ -76,39 +82,47 @@ public class DynamicRes : MonoBehaviour
 
-You can also override the upscale  options in the HDRP asset for each Camera in your scene using code.  To do this, call `DynamicResolutionHandler.SetUpscaleFilter(Camera camera, DynamicResUpscaleFilter filter)`, to make HDRP ignore the value in the HDRP Asset for a given camera.
+<a name="TAA_Notes"></a>
 
 ## Notes on Temporal Anti-Aliasing (TAA) Upscaling
 
-When you enable Temporal Anti-Aliasing (TAA) Upscaling, it replaces Temporal Anti-Aliasing in the pipeline. This means:
+When you enable **Temporal Anti-Aliasing (TAA) Upscaling**, it replaces Temporal Anti-Aliasing in the pipeline. This means:
 
-- Temporal Anti-Aliasing is the only the anti-aliasing method and no other post-process anti-aliasing option will work.
-- The post-process pipeline runs at final resolution. Before HDRP applies TAA Upscaling, the pipeline uses a downscaled resolution. This gives a more precise post-processing result, but also uses more GPU resources.
+- When TAA Upscaling is active, it is the only anti-aliasing method. No other post-process anti-aliasing option will work.
+- Before HDRP applies TAA Upscaling, the pipeline uses a downscaled resolution. This gives a more precise post-processing result, but also uses more GPU resources.
 
-Temporal Anti-Aliasing Upscaling sets the [Camera's](HDRP-Camera.md) **TAA Quality Preset** to **High Quality**. When you select the Temporal Anti-Aliasing Upscaling method, you cannot use change this preset.
+TAA Upscaling sets the [Camera's](HDRP-Camera.md) **TAA Quality Preset** to **High Quality**. When you enable the Temporal Anti-Aliasing (TAA) Upscaling, you cannot change this preset.
 
-Any option that can control TAA also controls TAA Upscaling. however it is important to remember that the source of current frame is lower resolution than the final image/history buffer. For this reason:
+Any option that can control TAA also controls TAA Upscaling. However, the source of the current frame is lower resolution than the final image/history buffer. For best results, take the following into account:
 
--  It is important to be cautious when setting up speed rejection thresholds. Speed rejection heavily reduces the influence of the history in favor of current frame, this mean that at lower screen percentages speed rejection can lead to a fairly low resolution looking image.
-- Given that the source data is low resolution, it is worth to explore the possibility of being more generous with the sharpening setting than what would be normally used at full resolution.
+- Speed rejection can produce an image that appears to have a low resolution at lower screen percentages. This is because the speed rejection threshold can reduce the influence of the history buffer in favor of the current frame.
+- You can set the [Camera's](HDRP-Camera.md) **TAA Sharpen Strength** setting to higher values to compensate for low resolution source data.
```

Comment:
This is way too definite, put it like this makes it sound is something that is always to do. It is something worth exploring w.r.t the usual range a user would use in TAA